### ISSUE TITLE
Plot of isomiRs/miRNA distribution using ggplot-python

### ISSUE THREAD

**Issue by lpantano** [miRTop/mirtop#21]
Implement option in `stats` sub-command like `--detail` to:

[ ] show distribution of sequences per sample (geom_density)
[ ] boxplot distribution per sample after normalization like DESeq2
[ ] summary figure like `isoPlot(type = "all")` from isomiRs bioc package

**Comment by cranewen** [comment]
Hi Lorena,

I have been working on the plot. I couldn't make the plot correctly in ggplot. It seems missing y axis values. But it works for dots or lines graphs. It looks like that. Please let me know if you have any idea on that.
<img width="748" alt="screen shot 2018-01-20 at 12 38 52 pm" src="https://user-images.githubusercontent.com/9448300/35186254-33e64cb4-fddf-11e7-81da-c52bf911b15b.png">

Or I can make it with Plotly, it looks like this:
![newplot](https://user-images.githubusercontent.com/9448300/35186219-aa0c5a60-fdde-11e7-8ef3-f1c399a58e72.png)

What do you think?

**Comment by lpantano** [comment]
Hi,

That seems a good format as well. But I see all of them looks like 1, is that a typo? If we the right numbers as it is in the example code, you can make a PR to the code adding the code inside the `mirtop/plot` folder.

Thanks a lot!

**Comment by cranewen** [comment]
Hi Lorena,

No, it's not a typo. It's the problem I am facing now. 
If I use line or point plot, it shows below, which is what I expected.
The code is ggplot(df, aes(x='tool',y='counts')) + geom_line() + facet_wrap('category')
<img width="766" alt="screen shot 2018-01-23 at 8 32 34 am" src="https://user-images.githubusercontent.com/9448300/35278883-622fac6e-0019-11e8-82d9-66534758088e.png">

But in the bar chart, ggplot(df, aes(x='tool',y='counts')) + geom_bar() + facet_wrap('category')
I don't think y='counts' works in the function. 

I will pull request once I fixed this problem. If you know what's wrong here, please let me know.

Thanks,
Crane

**Comment by lpantano** [comment]
Thanks for working on this.

Can you try to add this option inside geom_bar function: stat = “identity",

And see if that fixes the issue.

Cheers

**Comment by cranewen** [comment]
No, it's the same.

**Comment by lpantano** [comment]
Other option is to plot a point instead with geom_point and increase the size a little. We can try to figure out the issue later as well.

Thanks!

**Comment by cranewen** [comment]
Sure, Lorena. But I will try to find out a way to plot it in bar charts tonight. Do you want me to pull request? I also did column 9 of GFF and return key #25. I am gonna keep working on all the issues. I now have time during the nights, and I am learning the whole project slowly:)

**Comment by lpantano** [comment]
Yes, please, one pull request per issue. And please, update the REAMDE.md with your name in the contributors section. 

Thanks
